### DISCUSSION TITLE
Create Realistic Logging on and off

### DISCUSSION THREAD

**Discussion by dh4340** [cmu-sei/GHOSTS#435] [Ideas]
**Dynamic Logging in and off**: Introduce a dynamic system that allows for more flexibility and realism in logging on and off of systems, particularly in windows. Different NPCs should be able to log on to different Windows machines or Linux machines.

**Comment by sei-dupdyke** [comment]
We do this with a system we built that does this through vmware, but its not ready for prime time, sorry.
